src/core/Float.tsx

```diff
@@ -1,24 +1,31 @@
 import * as React from 'react'
 import { useFrame } from '@react-three/fiber'
 import mergeRefs from 'react-merge-refs'
+import * as THREE from 'three'
 
 export type FloatProps = JSX.IntrinsicElements['group'] & {
   speed?: number
   rotationIntensity?: number
   floatIntensity?: number
   children?: React.ReactNode
+  floatingRange?: [number | undefined | null, number | undefined | null]
 }
 
 export const Float = React.forwardRef<THREE.Group, FloatProps>(
-  ({ children, speed = 1, rotationIntensity = 1, floatIntensity = 1, ...props }, forwardRef) => {
+  (
+    { children, speed = 1, rotationIntensity = 1, floatIntensity = 1, floatingRange = [-1, 1], ...props },
```

Comment:
Let's have the default range be `[-Infinity, Infinity]` as to not cause breaking changes

Reply:
Wait if the expression only goes from [-0.1, 0.1] - would remapping it to [-Infinity, Infinity] cause issues?

Reply:
Ah yeah forgot this is not clamp. Anyways let’s make the default `+/- 0.1 * floatRange` So the default is such that there is no re-mapping

Reply:
Yea that's what I've done in the recent commit! the default remapping is [-0.1, 0.1].